packages/frontend/src/components/UserSettings/DeleteProjectPanel/DeleteProjectModal.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+    Button,
+    Group,
+    Modal,
+    ModalProps,
+    Stack,
+    Text,
+    TextInput,
+    Title,
+} from '@mantine/core';
+import { IconAlertCircle } from '@tabler/icons-react';
+import { FC, useState } from 'react';
+import { useProject } from '../../../hooks/useProject';
+import { useDeleteProjectMutation } from '../../../hooks/useProjects';
+import MantineIcon from '../../common/MantineIcon';
+
+export const ProjectDeleteModal: FC<
+    Pick<ModalProps, 'opened' | 'onClose'> & {
+        projectUuid: string;
+        onDelete: () => void;
+    }
+> = ({ opened, onClose, projectUuid, onDelete }) => {
+    const { isLoading, data: project } = useProject(projectUuid);
+    const { mutateAsync, isLoading: isDeleting } = useDeleteProjectMutation();
+
+    const [confirmOrgName, setConfirmOrgName] = useState<string>();
+
+    if (isLoading || !project) return null;
+
+    const handleConfirm = async () => {
+        await mutateAsync(projectUuid);
+        onClose();
+        onDelete();
+    };
+
+    const handleOnClose = () => {
+        setConfirmOrgName(undefined);
+        onClose();
+    };
+
+    return (
+        <Modal
+            size="md"
+            centered
+            opened={opened}
+            title={
+                <Group spacing="xs">
+                    <MantineIcon size="lg" icon={IconAlertCircle} color="red" />
+                    <Title order={4}>Delete Project</Title>
+                </Group>
+            }
+            onClose={handleOnClose}
+        >
+            <Stack>
+                <Text>
+                    Type the name of this Project <b>{project.name}</b> to
```

Comment:
I find <b> to be very heavy. instead you can use <Text inline fw={600}>{project.name}</Text>